Integration test (TestIntegrationAllConfigs) currently fails on main branch because of SLO validation

**Describe the bug**
See the following workflow run. The integration tests currently fail for the SLO config: https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/runs/4177060650?check_suite_focus=true

This might indicate a breaking change in the SLO API validation.

**How to reproduce**
The responsible configs are:
https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/tree/main/cmd/monaco/test-resources/integration-all-configs/project/slo

**Expected behavior**
The integration tests should work.

**Log output**
Thsi is the log output of the failing test:

```
=== RUN   TestIntegrationAllConfigs
You are currently using the old CLI structure which will be used by
default until monaco version 2.0.0

Check out the beta of the new CLI by adding the environment variable
  "NEW_CLI".

We can't wait for your feedback.

2021-11-11 10:59:52 INFO  Dynatrace Monitoring as Code v1.6.1
2021-11-11 10:59:52 WARN  You are using the configuration 'application', which will be deprecated in v2.0.0. Replace with type 'application-web'.
2021-11-11 10:59:52 INFO  Executing projects in this order: 
2021-11-11 10:59:52 INFO  	1: test-resources/integration-all-configs/project (30 configs)
2021-11-11 10:59:52 INFO  	2: test-resources/integration-all-configs/only-post (1 configs)
2021-11-11 10:59:52 INFO  Processing environment environment1...
2021-11-11 10:59:52 INFO  	Processing project test-resources/integration-all-configs/project...
2021-11-11 10:59:53 INFO  Deployment summary:
2021-11-11 10:59:53 ERROR Deployment to environment1 failed with error!
2021-11-11 10:59:53 ERROR 	Failed to create DT object My SLO_1636628392912940807AllConfigs (HTTP 400)!
    Response was: {"error":{"code":400,"message":"SLO validation failed with 1 error(s):\nNo Management Zone selected."}}, responsible config: test-resources/integration-all-configs/project/slo/slo.json
2021-11-11 10:59:53 ERROR Errors during deployment! Check log!
    all_configs_integration_test.go:44: assertion failed: 1 (statusCode int) != 0 (int)
--- FAIL: TestIntegrationAllConfigs (0.95s)
```

**Environment (please complete the following information):**
All environments in CI

--- a/cmd/monaco/test-resources/integration-all-configs/project/slo/slo.json
+++ b/cmd/monaco/test-resources/integration-all-configs/project/slo/slo.json
@@ -6,7 +6,7 @@
   "metricNumerator": "builtin:service.errors.server.successCount",
   "metricDenominator": "builtin:service.requestCount.total",
   "evaluationType": "AGGREGATE",
-  "filter": "type(\"HOST\")",
+  "filter": "mzName(\"{{ .mzone }}\"),type(\"HOST\")",
   "target": 95,
   "warning": 97.5,
   "timeframe": "-1d"

--- a/cmd/monaco/test-resources/integration-all-configs/project/slo/slo.yaml
+++ b/cmd/monaco/test-resources/integration-all-configs/project/slo/slo.yaml
@@ -3,3 +3,4 @@ config:
 
 demo_slo:
   - name: "My SLO"
+  - mzone: "/project/management-zone/zone.name"

